### PR TITLE
Add Hogfish validator

### PR DESCRIPTION
**Pull request by webmaster128**
Please review carefully

### CODE REVIEW

File: coral/config/gentx/gentx-hogfish.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            "max_rate": "0.200000000000000000",
+            "max_change_rate": "0.010000000000000000"
+          },
+          "min_self_delegation": "1",
+          "delegator_address": "coral10zn0d2eeust0495crtr3zqz7t688hg0s53afrh",
+          "validator_address": "coralvaloper10zn0d2eeust0495crtr3zqz7t688hg0sm7wr4f",
+          "pubkey": "coralvalconspub1zcjduepqvxg72ccnl9r65fv0wn3amlk4sfzqfe2k36l073kjx2qyaf6sk23qw7j8wq",
+          "value": {
+            "denom": "ureef",
+            "amount": "1000000"
```
Comment (ethanfrey) [inline coral/config/gentx/gentx-hogfish.json:26]: Only starting with 1 REEF, not all 100?
Still, should be enough to start up.

File: coral/config/gentx/gentx-hogfish.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            "security_contact": "",
+            "details": ""
+          },
+          "commission": {
+            "rate": "0.100000000000000000",
+            "max_rate": "0.200000000000000000",
+            "max_change_rate": "0.010000000000000000"
+          },
+          "min_self_delegation": "1",
+          "delegator_address": "coral10zn0d2eeust0495crtr3zqz7t688hg0s53afrh",
```
Comment (ethanfrey) [inline coral/config/gentx/gentx-hogfish.json:21]: Yes, recognize this address

**Review by ethanfrey — APPROVED** [review]
Okay, welcome as the seed validator.

File: coral/config/gentx/gentx-hogfish.json (hunk 1 above)
Comment (webmaster128) [reply coral/config/gentx/gentx-hogfish.json:26]: This is the amount documented at https://docs.cosmwasm.com/testnets/testnets.html#become-an-active-validator-optional

I am aware I have more, but if I understand correctly, it should be enough

**Pull request merged**